File: 0169-majority-element/0169-majority-element.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int j=0;
        int count = 1;
        for(int i = 1; i < nums.size(); i++){
            if ( nums[j] == nums[i]){
                count += 1;
                if (count > nums.size()/2){
                    return nums[j];
                }
            }
            else{
                j=i;
                count = 1;
            }
        }
        return nums[j];
        
    }
};

File: 0169-majority-element/0169-majority-element.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        nums.sort()
        j = 0
        count = 1
        for i in range(1, len(nums)):
            if nums[j] == nums[i]:
                count += 1
                if count > len(nums)/2:
                    return nums[j]

            else:
                j = i
                count = 1

        return nums[j]
        